Automattic/wp-calypso
# A4A: Customize login screen in connection flow

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/48

## Proposed Changes

* Customizes the login block when the user is coming from the A4A client plugin:
  * Use the Automattic for Agencies logo.
  * Hide site-only connection options.

## Testing Instructions



* While signed out, in Calypso, go to `/log-in/jetpack?from=automattic-for-agencies-client`
* Validate the automattic for agencies logo is displayed.
* Validate there is **no** "Or start using Jetpack now" blurb displayed at the very bottom of the page, under the login buttons.

## Pre-merge Checklist





## Screenshots

<img width="549" alt="Screenshot 2024-04-15 at 2 10 44 PM" src="https://github.com/Automattic/wp-calypso/assets/10933065/3f6262f2-5670-47c6-b2b5-7694d5d140fb">
